### ISSUE TITLE
simulation blows up when switching on attenuation

### ISSUE THREAD

**Issue by wolfjonathan** [kuangdai/AxiSEM-3D#15]
Hi Kuangdai,

There is one issue that I am not able to solve: When I run a simulation using a 3D volumetric model specifying VS using a certain Axisem mesh, no error occurs. Then, when I switch on CG4 or full attenuation (and keep all the other input parameters the same), the simulation crashes at the second time step:
FROM: Domain::checkStability
WHAT: Simulation has blown up, with ΔT = 0.02582 s.
      Where the instability occurred:
      * (r, θ)  =  (3.48e+06, 2.88646)
      When the instability occurred:
      * current time  =  -3.97628
      * current step  =  2

What I have tried so far:
- The error message suggested to me that somehow the mesh was flawed, so I tried other meshes for which the simulation also blew up at time step 2 (using the 3D model + attenuation). 
- Then I deactivated the 3D model, keeping attenuation switched on and keeping the same mesh. The simulation does not crash.
- I played with certain parameters, letting Axisem choose the time step, I lowered the Courant no to 0.3, tried switching on both or only one absorbing boundary ... but still the simulation crashed at time step 2 when including the 3D volumetric model + attenuation.

In short: I can run simulations with (a mesh + attenuation) or (a mesh + a certain 3D volumetric model) but not (a mesh + a certain 3D volumetric model + attenuation).

Do have an idea what could be causing this issue?

Best,

Jonathan

**Comment by wolfjonathan** [comment]
Update: This issue consistently occurs with different meshes and 3D volumetric models. Mesh + 3D volumetric model + Attenuation fails, although Mesh + 3D volumetric model and Mesh + Attenuation works.

**Comment by kuangdai** [comment]
Hi Jonathan, please provide the complete input folder that triggers the blow-up.

**Comment by wolfjonathan** [comment]
Thanks, Kuangdai. I have given you access to a google drive folder (because the original folder is too large to upoad here), and will upload another 'small as possible' (just saw that now) folder here in a bit after a meeting so that everyone can follow the problem.

**Comment by wolfjonathan** [comment]
[test1.zip](https://github.com/kuangdai/AxiSEM-3D/files/6050352/test1.zip)
This is small and reproduces it.

**Comment by kuangdai** [comment]
At what step does instability occur for this small test?  On my machine, it runs well. Can you upload the standard output of  this small test.

**Comment by wolfjonathan** [comment]
Great that it works on your machine! It's time step 2, see attached. It's always time step 2, also for other combinations of meshes and 3D models. I am using a newly downloaded and built version of AxiSEM3D (8 hours ago)
[output.txt](https://github.com/kuangdai/AxiSEM-3D/files/6050866/output.txt)

**Comment by wolfjonathan** [comment]
Just in case this helps: Attached are the outputs for two identical simulations. The only difference is that I switched on attenuation for one of them. The one without attenuation runs without problems and I cancelled manually after it had successfully started the loop.

[output_att.txt](https://github.com/kuangdai/AxiSEM-3D/files/6051006/output_att.txt)
[output_no_att.txt](https://github.com/kuangdai/AxiSEM-3D/files/6051009/output_no_att.txt)

**Comment by kuangdai** [comment]
This bug is fixed. Thanks.

**Comment by wolfjonathan** [comment]
Thank you. Runs flawlessly now!
